docs/ecosystem/panel.md

```diff
@@ -1,108 +1,221 @@
-    data = gdf[gdf["state"] == state]
-    layer = PathLayer.from_geopandas(data, width_min_pixels=0.8)
-    normalized_scale_rank = (data["scalerank"] - 3) / 9
-    layer.get_color = apply_continuous_cmap(normalized_scale_rank, palette, alpha=alpha)
-    map_ = Map(layer, _height=650)
-    return map_
+With Panel, you can build reusable components and styled web applications in just a few lines of code.
 
+The example below is based on the [North America Roads](../examples/north-america-roads.ipynb) example.
 
-description = """# lonboard
+![Panel lonboard application](../assets/panel-application-example.gif)
 
-A Python library for **fast, interactive geospatial vector data visualization** in Jupyter (and Panel).
+```python
+import colorcet as cc
+import geopandas as gpd
+import param
 
-By utilizing new technologies like `GeoArrow` and `GeoParquet` in conjunction with GPU-based map rendering, lonboard aims to enable visualizing large geospatial datasets interactively through a simple interface."""
+from lonboard import Map, PathLayer
+from lonboard.colormap import apply_continuous_cmap
+from lonboard._viewport import compute_view
+from palettable.palette import Palette
 
+import panel as pn
 
-# THE PANEL APP
 pn.extension("ipywidgets")
-state = pn.widgets.Select(
-    value="California",
-    options=state_options,
-    width=150,
-    name="State",
-    sizing_mode="stretch_width",
-)
-cmap = pn.widgets.ColorMap(
-    value=cc.fire,
-    options=cc.palette,
-    ncols=3,
-    swatch_width=100,
-    name="cmap by Colorcet",
-    sizing_mode="stretch_width",
-)
-alpha = pn.widgets.FloatSlider(
-    value=0.8, start=0, end=1, name="Alpha", min_width=100, sizing_mode="stretch_width"
+
+url = "https://naciscdn.org/naturalearth/10m/cultural/ne_10m_roads_north_america.zip"
+gdf = pn.state.as_cached(
+    "ne_10m_roads_north_america", gpd.read_file, filename=url, engine="pyogrio"
 )
+
+state_options = sorted(state for state in gdf["state"].unique() if state)
+
+description = """# Lonboard
+
+A Python library for **fast, interactive geospatial vector data visualization** in Jupyter (and Panel).
+
+By utilizing new technologies like `GeoArrow` and `GeoParquet` in conjunction with GPU-based map rendering, Lonboard aims to enable visualizing large geospatial datasets interactively through a simple interface."""
+
 logo = pn.pane.Image(
     "https://github.com/developmentseed/lonboard/raw/main/assets/dalle-lonboard.jpg"
 )
-def title(state):
-    return f"# North America Roads: {state}"
-
-settings = pn.Column(state, cmap, alpha)
-description = pn.Column(pn.pane.Markdown(description, margin=5), logo)
-component = pn.Column(
-    pn.bind(title, state=state),
-    pn.panel(
-        pn.bind(create_map, state=state, cmap=cmap, alpha=alpha.param.value_throttled),
-        sizing_mode="stretch_both",
-    ),
-    sizing_mode="stretch_both",
-)
+
+def to_rgb(hex: str) -> list:
+    h = hex.strip("#")
+    return list(int(h[i : i + 2], 16) for i in (0, 2, 4))
```

Comment:
Possibly a _slight_ preference for suggesting users to use [`matplotlib.colors.to_rgb`](https://matplotlib.org/stable/api/_as_gen/matplotlib.colors.to_rgb.html) instead of rolling their own hex -> RGB conversion, as we do internally [here](https://github.com/developmentseed/lonboard/blob/47464d54042e9bf7889fc6854b3b82fb1d33af5a/lonboard/traits.py#L309), but if `Palette` needs integers instead of floats then we might as well keep it as is.

Reply:
I tried and I don't get a colormap. I just get a black color.

![image](https://github.com/developmentseed/lonboard/assets/42288570/bc683fb0-632b-47b7-9cc9-49dfd81f1cd6)